src/ALE/MOM_ALE.F90

```diff
@@ -753,8 +753,10 @@ subroutine ALE_regrid_accelerated(CS, G, GV, h, tv, n, u, v, OBC, Reg, dt, dzReg
 
     ! remap from original grid onto new grid
     do j = G%jsc-1,G%jec+1 ; do i = G%isc-1,G%iec+1
-      call remapping_core_h(CS%remapCS, nz, h_orig(i,j,:), tv%S(i,j,:), nz, h(i,j,:), tv_local%S(i,j,:))
-      call remapping_core_h(CS%remapCS, nz, h_orig(i,j,:), tv%T(i,j,:), nz, h(i,j,:), tv_local%T(i,j,:))
+      call remapping_core_h(CS%remapCS, nz, h_orig(i,j,:), tv%S(i,j,:), nz, h(i,j,:), tv_local%S(i,j,:), &
```

Comment:
I agree that this will pass the dimensional rescaling tests, and that this is what we will eventually want to use.  However, it will change answers if this code is exercised in any existing test cases.  One way to both correct the rescaling and preserve the old answers would be to follow the pattern from lines 375-381 and 401-404 of MOM_diag_remap.F90:

  if (.not.remap_cs%answers_2018) then
    h_neglect = GV%H_subroundoff ; h_neglect_edge = GV%H_subroundoff
  elseif (GV%Boussinesq) then
    h_neglect = GV%m_to_H*1.0e-30 ; h_neglect_edge = GV%m_to_H*1.0e-10
  else
    h_neglect = GV%kg_m2_to_H*1.0e-30 ; h_neglect_edge = GV%kg_m2_to_H*1.0e-10
  endif

   ...

   call remapping_core_h(..., h_neglect, h_neglect_edge)

I don't think that anyone at GFDL will care if we change these answers (and I prefer the simpler form you propose), but someone else might care about answer changes in some of these cases.